src/components/section/index.tsx

```diff
@@ -1,29 +1,52 @@
-import React, { FC, ReactNode } from 'react';
+import React, { ElementType, FC, ReactNode } from 'react';
 
-import { Box, Heading, Text } from '@chakra-ui/react';
+import { Box, chakra, useMultiStyleConfig } from '@chakra-ui/react';
 
 import Stack from '../stack';
 
 interface Props {
-  variant: 'hero';
+  variant?: 'hero' | 'regular';
   title: string;
   text?: string;
   image?: () => ReactNode;
+  as?: ElementType;
 }
 
-const Section: FC<Props> = ({ title, text, image, variant }) => {
+const Section: FC<Props> = ({
+  as = chakra.div,
+  title,
+  text,
+  image,
+  variant,
+  ...rest
+}) => {
+  const styles = useMultiStyleConfig('Section');
+  const Component = chakra(as, {
+    baseStyle: styles.section,
+  });
+
+  const titleStyle = {
+    ...(variant === 'hero' ? styles.titleHero : styles.titleRegular),
+  };
+
+  const textStyle = {
+    ...(variant === 'hero' ? styles.textHero : styles.textRegular),
+  };
+
   return (
-    <Stack
-      direction={{ xs: 'column', lg: 'row' }}
-      spacing="xl"
-      align={{ xs: 'center', lg: 'baseline' }}
-    >
-      {variant === 'hero' && image ? <Box>{image()}</Box> : null}
-      <Stack spacing="md">
-        <Heading textStyle="heading1">{title}</Heading>
-        {text ? <Text textStyle="body-large">{text}</Text> : null}
+    <Component {...rest}>
+      <Stack
+        direction={{ xs: 'column', lg: 'row' }}
+        spacing="xl"
+        align={{ xs: 'center', lg: 'baseline' }}
+      >
+        {variant === 'hero' && image ? <Box>{image()}</Box> : null}
```

Comment:
@Averethel you had this question -> `Why wouldn't we show the image for a non-hero section?`

After checking with Enrique: there will be cases where in some pages we won't show and image and we want to be flexible on this point.